src/views/Dashboard/Lab/Services/Add.vue

```diff
@@ -282,6 +298,14 @@ export default {
 
   },
 
+  async created() {
+    const { verification_status, info: { email } } = await queryLabsById(this.api, this.pair.address)
+    if (verification_status && verification_status !== "Unverified") return
```

Comment:
Instead of checking if it's unverified isn't it better to be: `verification_status === "Verified"`?